app/controllers/util/DynamicFormUtils.scala

```diff
@@ -11,6 +11,7 @@ class DynamicFormUtils(request: Request[AnyContent], defaultFieldValues: List[Fo
   lazy val formAnswersWithValidInputNames: Map[String, Seq[String]] = formAnswers.filter {
     case (inputName, _) if inputName.startsWith("input") => true
     case (inputName, _) if inputName == "csrfToken"      => false
+    case (inputName, _) if inputName.startsWith("tna-")  => false
```

Comment:
Is there a reason this input is being excluded? Does the name need to be more specific?